kinder/pkg/kubeadm/config.go

```diff
@@ -66,14 +60,15 @@ func Config(kubeadmVersion *K8sVersion.Version, data ConfigData) (config string,
 	return buff.String(), nil
 }
 
-// getKubeadmConfigVersion returns the kubeadm config version corresponding to a Kubernetes kubeadmVersion
-func getKubeadmConfigVersion(kubeadmVersion *K8sVersion.Version) (string, error) {
-	// returns the corresponding config version
-	// nb v1alpha1 (that is Kubernetes v1.10.0) is out of support,
-	// v1alpha2 (that is Kuberntes v1.11.0) is out of support.
-	// v1alpha3 (that is Kuberntes v1.13.0) is out of support.
-
-	return "v1beta3", nil
+// GetKubeadmConfigVersion returns the kubeadm config version corresponding to a Kubernetes kubeadmVersion
+func GetKubeadmConfigVersion(kubeadmVersion *K8sVersion.Version) string {
+	// v1alpha1 (that is Kubernetes v1.10.0) is out of support
+	// v1alpha2 (that is Kubernetes v1.11.0) is out of support
+	// v1alpha3 (that is Kubernetes v1.13.0) is out of support
+	if kubeadmVersion.Major() > 1 || kubeadmVersion.Minor() >= 22 {
```

Comment:
```suggestion
	if kubeadmVersion.Compare("v1.22.0") >= 0 {
```

you can do this, which is technically the accurate check.
but the accuracy does not matter much since we have only MAJOR == 1 for k8s.
https://github.com/kubernetes/apimachinery/blob/master/pkg/util/version/version.go#L319